server/database/default.json

```diff
@@ -1,3 +1,3 @@
 {
-  "ConnectionUrl": "mongodb+srv://greg34910:cookie123@cluster0-pf0b2.mongodb.net/memestagram?retryWrites=true&w=majority"
+  "ConnectionUrl": "mongodb+srv://drunkenclam123:gregy624@cluster0-pf0b2.mongodb.net/memestagram?retryWrites=true&w=majority"
```

Comment:
Could you also make this a generic instead of using your own account?

Reply:
I will change this also